client/src/auth/slice.js

```diff
@@ -5,6 +5,7 @@ export const initialState = {
     id: '',
     name: '',
   },
+  activeGames: {},
```

Comment:
Hmm I'm tryna think the best way to define the state here. Do we really want active games to live on the auth state? Maybe, because so far the auth state is holding all info about the logged in player. But it doesn't seem ideal.

Reply:
let's move to a "home" saga/slice/state.